tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -357,6 +357,96 @@ test.describe( 'Set up accounts', () => {
 				).toBeVisible();
 			} );
 		} );
+
+		test.describe( 'Claim Google Ads Account adsaccountclaim', () => {
+			test.beforeEach( async () => {
+				await setUpAccountsPage.mockJetpackConnected();
+				await setUpAccountsPage.mockGoogleConnected();
+				await setUpAccountsPage.mockMCConnected();
+
+				await setUpAccountsPage.fulfillAdsConnection( {
+					id: 12345,
+					currency: 'USD',
+					status: 'incomplete',
+					step: 'account_access',
+					sub_account: true,
+					symbol: '$',
+				} );
```

Comment:
Let's turn this into a mock, similar to `mockAdsAccountIncomplete()` ([ref](https://github.com/woocommerce/google-listings-and-ads/blob/599c2d209707ce06f382fd82e0c181bd04543e46/tests/e2e/utils/mock-requests.js#L577-L590)).

```js
	/**
	 * Mock Google Ads account as connected but unclaimed.
	 *
	 * @return {Promise<void>}
	 */
	async mockAdsAccountUnclaimed( id = 12345 ) {
		await this.fulfillAdsConnection( {
			id,
			currency: 'TWD',
			symbol: 'NT$',
			status: 'incomplete',
			step: 'billing',
		} );
	}
```